send-client/src/test/java/ai/apptuit/metrics/client/ApptuitPutClientTest.java

```diff
@@ -139,6 +135,37 @@ public void testPut400() throws Exception {
     testPut(400);
   }
 
+  @Test
+  public void testSend200() throws Exception {
+    testSend(200, false);
+  }
+
+  @Test
+  public void testSendFakeToken() throws Exception {
+    testSend(401, true, "FAKE_TOKEN");
+  }
+
+  @Test
+  public void testSend400() throws Exception {
+    testSend(400, true);
+  }
+
+  @Test
+  public void testSend500() throws Exception {
+    testSend(500, true);
+  }
+
+  @Test
+  public void testSendConnectionError() throws MalformedURLException, ParseException {
+    String url = "http://localhost:" + 123 +"/api/put";
+    try {
+      testSend(false, MockServer.token, new URL(url));
+      assert(false);
+    } catch (ConnectException e) {
```

Comment:
Remove the catch and change the method annotation to `@Test(expected = ConnectException.class)`